teach/templates/clubs/email/create_club_staff/body.txt

```diff
@@ -0,0 +1,14 @@
+Club captain {{ full_name }} would like to add their Club to the map (owned by {{ username }}).
```

Comment:
this isn't what's happening though. `full_name` is filing an application to become a Mozilla club captain. They're a club captain yet. They also don't want to get their club added to the map, they want their club application approved, which has a side effect of being on the map.

```
{{ full_name }} ({{ username }}) has filled in the Club Captain Application form with the following data:
```
would be more accurate.